controllers/quickstart_constants.go

```diff
@@ -238,6 +238,54 @@ spec:
         failedTaskHelp: This task isn’t verified yet. Try the task again.
       summary:
         success: You have successfully created BucketClass
+        failed: Try the steps again. 
+    - title: Encrypt your StorageClass
+      description: |-
+
+          StorageClass encryption enables you to provide individual keys to each of your StorageClusters.
+
+
+          You can create an encrypted StorageClass while configuring your StorageCluster, or when creating a new StorageClass.
+
+
+          **To create an encrypted StorageClass while setting up your cluster:**
+
+          1. After installing OpenShift Data Foundation, select **Create StorageSystem**.
```

Comment:
Path to Openshift Data Foundation ? We should probably guide them to the operators

Reply:
In the navigation menu, select **Storage > OpenShift Data Foundation**.

Reply:
We have already documented these steps in "Install OpenShift Data Foundation" guide. Adding it here as well will unnecessarily add extra steps to the page.
We will have to add these steps again, if we want to tell user how to create a StorageSystem which IMO is might not be required. Plz let me know your thoughts.
1) In the main navigation menu, select  and select Installed Operators.
2) From your list of installed operators, select OpenShift Data Foundation.
3) On the StorageSystem tile, select Create Instance.